* 2009 Nov 22		Kevin Quick		<quick@sparq.org>

    The test directories should not be links, but "/bin" and "/usr/bin"
    are links under GoboLinux.  Use alternates.

--- hg-src/src/pkg/os/os_test.go	2009-11-13 22:01:14.000000000 -0700
+++ /Files/Compile/Sources/Golang-hg/src/pkg/os/os_test.go	2009-11-22 14:12:20.000000000 -0700
@@ -467,7 +467,7 @@
 	}
 	// These are chosen carefully not to be symlinks on a Mac
 	// (unlike, say, /var, /etc, and /tmp).
-	dirs := []string{"/bin", "/", "/usr/bin"};
+	dirs := []string{"/System", "/", "/Programs"};
 	for mode := 0; mode < 2; mode++ {
 		for _, d := range dirs {
 			if mode == 0 {
